File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/presenter/CapaPresenter.java
package br.com.levimendesestudos.spidermagazine.mvp.presenter;

import br.com.levimendesestudos.spidermagazine.mvp.contracts.CapaMVP;

/**
 * Created by 809778 on 22/08/2017.
 */

public class CapaPresenter extends CapaMVP.Presenter {

    CapaMVP.View mView;

    public CapaPresenter(CapaMVP.View view) {
        mView = view;
    }

    @Override
    public void init() {
        mView.carregarCapa();
    }
}

File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/presenter/CapaPresenter.kt
package br.com.levimendesestudos.spidermagazine.mvp.presenter

import br.com.levimendesestudos.spidermagazine.mvp.contracts.CapaMVP

/**
 * Created by 809778 on 22/08/2017.
 */

class CapaPresenter(private val mView: CapaMVP.View) : CapaMVP.Presenter {

    override fun init() {
        mView.carregarCapa()
    }
}